README.md

```diff
@@ -50,3 +50,10 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+To start my code runs through loading an array n long with the numbers before it randomly scrambles them into the starting order so it goes through n twice. then we do the 2Opt section and we create a new array in 2Opt so worst case we would reverse the entire tour meaning we have to go through n once and we then need to calculate the distance of each one adding another n. The do-while loop goes 200 times so it would add O(200n). So worst case time complextiy would be O(n^2) + O(n^2) + O(200n) making the time complexity O(n^2).
```

Comment:
I don't know what you mean by "20pt". The number of iterations of the loop seems to depend on whether improvements are found.